Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -79,6 +79,27 @@ open class ReadingThemesControlsViewController: UIViewController, AnalyticsConte
         NSObject.cancelPreviousPerformRequests(withTarget: self)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(_:)))
+        slider.addGestureRecognizer(tap)
+    }
+    
+    func handleTap(_ sender: UIGestureRecognizer) {
+        let pointTapped: CGPoint = sender.location(in: self.view)
+        
+        let positionOfSlider: CGPoint = slider.frame.origin
+        let widthOfSlider: CGFloat = slider.frame.size.width
```

Comment:
if you change to `location(in: slider)` above, the resulting point is in the slider's view coordinates, so you would use `slider.bounds` since `slider.frame` is in the main view's coordinates